phpdotnet/phd/Package/Generic/XHTML.php

```diff
@@ -1811,7 +1811,7 @@ private function convertConstantNameToId(string $constantName): string {
         $tempLinkValue = str_replace(
```

Comment:
In a future version, move all transformations inside the normalize function (trim, replace), to make understanding the code easier.